.github/actions/nimbus-build-system/action.yml

```diff
@@ -9,9 +9,6 @@ inputs:
   cpu:
     description: "CPU to build for"
     default: "amd64"
-  nim_version:
```

Comment:
we should not remove this

Reply:
I argue in my doc that this is not really achieving much atm. While I agree that building against a compiler matrix is useful, what this currently does is pin the CI to a single version, and it's the wrong one/has to be kept in sync with the rest. :-)